core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -197,14 +207,7 @@ class ZkUtils(val zkClient: ZkClient,
                               BrokerSequenceIdPath,
                               IsrChangeNotificationPath)
 
-  val securePersistentZkPaths = Seq(BrokerIdsPath,
```

Comment:
Some output of the fixed function running on my cluster:
```
[2017-03-08 22:28:15,087] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/42/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,088] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/41/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,088] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/40/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,088] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/49/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,088] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/48/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,088] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/45/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,089] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/44/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,089] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/47/state (kafka.admin.ZkSecurityMigrator)
[2017-03-08 22:28:15,089] INFO Successfully set ACLs for /brokers/topics/__consumer_offsets/partitions/46/state (kafka.admin.ZkSecurityMigrator)
[201
```

As you can see I only provided the `/brokers` path in my code and the existing code explored this node recursively